cmdeploy/src/cmdeploy/cmdeploy.py

```diff
@@ -331,8 +361,9 @@ def main(args=None):
         return parser.parse_args(["-h"])
 
     def get_sshexec():
-        print(f"[ssh] login to {args.config.mail_domain}")
-        return SSHExec(args.config.mail_domain, verbose=args.verbose)
+        host = args.ssh_host if hasattr(args, "ssh_host") and args.ssh_host else args.config.mail_domain
+        print(f"[ssh] login to {host}")
+        return SSHExec(host, verbose=args.verbose)
```

Comment:
This is a good step towards getting `cmdeploy dns` to run as well... but maybe we can completely avoid logging in to localhost? That would be much simpler. Let's see, for now at least deploying works, after all :)